src/bundle/SystemInfo/Collector/IbexaSystemInfoCollector.php

```diff
@@ -34,7 +34,8 @@ class IbexaSystemInfoCollector implements SystemInfoCollector
         '2.5' => '2019-03-29T16:59:59+00:00',
         '3.0' => '2020-04-02T23:59:59+00:00',
         '3.1' => '2020-07-15T23:59:59+00:00',
-        '3.2' => '2020-09-20T23:59:59+00:00', // Estimate at time of writing
+        '3.2' => '2020-10-23T23:59:59+00:00',
+        '3.3' => '2020-12-30T23:59:59+00:00', // Estimate at time of writing
```

Comment:
It will be before this (due to xmas), check for planned date here maybe

Reply:
I was being pessimistic on purpose here, but LS confirms, so 🤷‍♂️